### PR TITLE
Likes  history

### PR DESCRIPTION
**Pull request by Manpreet-hub**
This PR contains:-

-  like/dislike functionality
-  History page functionality
-  Added horizontal video & react player component

### CODE REVIEW

File: src/services/likeVideoService.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            authorization: token,
+          },
+        }
+      );
+      dataDispatch({ type: "ADD_TO_LIKE", payload: likes });
+    } catch (error) {
+      console.log(error);
+    }
+  } else alert("Please login first");
+};
```
Comment (ajay-mandaviya) [inline src/services/likeVideoService.js:25]: just used any react toast library for show alert

**Review by ajay-mandaviya — APPROVED** [review]
Everything looks good and well structure 
good to merge

File: src/services/historyService.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  } catch (error) {
+    console.log(error);
+  }
+};
+
+const postHistory = async (dataDispatch, video) => {
+  const token = localStorage.getItem("token");
+
+  try {
+    console.log("post history");
```
Comment (ashishnnnnn) [inline src/services/historyService.js:24]: Console can be removed .. if not required..

**Review by ashishnnnnn — APPROVED** [review]
Code looks clean ..
Can be merged..

File: src/services/likeVideoService.js (hunk 1 above)
Comment (Manpreet-hub) [reply src/services/likeVideoService.js:25]: yes, will add toast in the next PR.

File: src/services/historyService.js (hunk 2 above)
Comment (Manpreet-hub) [reply src/services/historyService.js:24]: ok

**Pull request merged**